swagger.yaml

```diff
@@ -1767,6 +1767,38 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /deleted/clusters:
```

Comment:
This would be better as a filter on list-clusters (boolean show_deleted) rather than a new API

Reply:
+1